ow/launch/common.launch

```diff
@@ -69,16 +69,24 @@
   <!-- Start faults node -->
   <node name="faults" pkg="ow_faults" type="faults" output="screen"/>
 
+  <!-- ====== simulate changes to the terrain caused by the scoop   ======= -->
+  <node name="modify_terrain_scoop" pkg="ow_dynamic_terrain" type="modify_terrain_scoop_pub.py" output="screen"/>
+
+  <!-- ====== simulate changes to the terrain caused by the grinder ======= -->
+  <node name="modify_terrain_grinder" pkg="ow_dynamic_terrain" type="modify_terrain_grinder_pub.py" output="screen"/>
+
```

Comment:
These two nodes enable the dynamic_terrain. Terry moved them from the europa_terminator_workspace.launch to common.launch such that it is turned on for all of the terrains

Reply:
Oh, I see! Thanks!

Reply:
We want to be able to dig in any terrain, so these belong in common.launch instead of a launch file for a specific gazebo world. In practice, digging only appears to work on terminator_workspace and atacama because they have high-resolution meshes. The other terrains are too course.

Reply:
Ok, thanks for the explanation. And yes, I usually realize there is a new comment because I get an email!